data.md

```diff
@@ -0,0 +1,4 @@
+## Data Retention Policy
+Our BigBlueButton instance keeps your uploaded presentations for 5 days after you have ended your meeting. BigBlueButton log history are kept for 28 days. Currently our BigBlueButton server is not backed up.
+
+Our Jitsi instance we keep standard system logs for debugging purpose only.
```

Comment:
How long?
Are they backed up?
What info might exist in these logs? 
e.g. 
- IP address
- Timestamp of visit
...?

Reply:
@ASoTNetworks please refer to https://jitsi.org/meet-jit-si-privacy/ to see what is relevant to us here.